openlibrary/components/AuthorIdentifiers.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <table>
+    <tr>
+      <th>
+        <select class="form-control" v-model="selectedIdentifier" name="name">
+          <option disabled value="">Select one</option>
+          <option v-for="idConfig in identifierConfigsByKey" :key="idConfig.name" :value="idConfig.name">
+            {{idConfig.label}}
+          </option>
+        </select>
+      </th>
+      <th>
+        <input class="form-control" type="text" name="value" id="id-value" v-model="inputValue" @keyup.enter=setIdentifier>
+      </th>
+      <th>
+        <button class="form-control" type="button" name="set" :disabled="!setButtonEnabled" @click=setIdentifier>Set</button>
+      </th>
+    </tr>
+    <tr :key="identifier.name" v-for="identifier in assignedIdentifiersWithConfigs">
```

Comment:
Also add a `v-if="value"`; we don't want rows without identifiers to show up.

Reply:
I don't think that will work while we are using Vue 2. https://v3.vuejs.org/guide/migration/v-if-v-for.html#_2-x-syntax

They recommend using a computed property.

Reply:
The way they've written the docs there is super confusing :P But I think in Vue 3 this will no longer be possible; Vue 2 should support it: https://vuejs.org/v2/guide/list.html#v-for-with-v-if

Grrr, that's annoying change in Vue 3 :/ Let's use it here for now, since we're still on Vue 2. I use this pattern _profusely_ throughout library explorer 😅